### PR TITLE
Allow to install helm charts via helm addon

### PR DESCRIPTION
**Pull request by jakolehm**
```yaml
  helm:
    enabled: true
    charts:
      - name: stable/prometheus
        repo: https://kubernetes-charts.storage.googleapis.com/
        values: ./prometheus.yml
```

### CODE REVIEW

**Comment by nevalla** [comment]
How about different repos?

**Comment by jakolehm** [comment]
> How about different repos?

Each chart can configure own repo.

File: addons/helm/addon.rb [hunk 1]
```diff
@@ -3,4 +3,180 @@
...
+              'helm.pharos.sh/chart': chart.name.split('/').last
+            }
+          },
+          spec: {
+            restartPolicy: "OnFailure",
+            serviceAccountName: "tiller",
+            containers: [
+              {
+                name: "helm",
+                image: "quay.io/kontena/pharos-helm-worker-amd64:2.12.3",
```
Comment (jakolehm) [inline addons/helm/addon.rb]: This needs to be fixed once we have images in CDN.

**Comment by jakolehm** [comment]
@nevalla @jnummelin PTAL

File: addons/helm/addon.rb [hunk 2]
```diff
@@ -1,6 +1,184 @@
...
+  # @return [K8s::Resource]
+  def build_job(chart, configmap)
+    K8s::Resource.new(
+      apiVersion: "batch/v1",
+      kind: "Job",
+      metadata: {
+        name: "helm-apply-#{chart.name.split('/').last}",
+        namespace: "kube-system",
+        labels: {
+          LABEL_NAME => chart.name.split('/').last
```
[the remark this answers is not in the document]
Comment (jakolehm) [reply addons/helm/addon.rb]: Fixed.

File: addons/helm/addon.rb [hunk 3]
```diff
@@ -3,4 +3,180 @@
...
+      kind: "Job",
+      metadata: {
+        name: "helm-apply-#{chart.name.split('/').last}",
+        namespace: "kube-system",
+        labels: {
+          'helm.pharos.sh/chart': chart.name.split('/').last
+        }
+      },
+      spec: {
+        backoffLimit: 1000,
```
[the remark this answers is not in the document]
Comment (jakolehm) [reply addons/helm/addon.rb]: Idea is that job should retry forever ... but I can lower it if it's too scary 😄

**Pull request merged**